sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/grafana.yaml

```diff
@@ -0,0 +1,100 @@
+provider:
+  metadata:
+    name: Grafana
+    category: Data visualization
+    message: Make sure this Grafana token gets revoked, changed, and removed from the code.
+  detection:
+    post:
+      patternNot:
+        - "(?i)(?:s|ex)ample|foo|bar|test|abcd|redacted"
+
+  rules:
+    - id: grafana-api-token # Note: deprecated
+      rspecKey: S6762
+      metadata:
+        name: Grafana API tokens should not be disclosed
+      detection:
+        pre:
+          include:
+            content:
+              # This secret pattern checks for the prefix "eyJrIjoi" which is `{"k":"` B64 encoded.
+              # The chance that this can be found in another secret or even normal data is
+              # still somewhat likely, so to be sure we filter for "grafana" here.
+              - grafana
+        matching:
+          pattern: "\\b(eyJrIjoi[a-zA-Z0-9]+(?:={1,2}\\B|\\b))"
+      examples:
+        - text: |
+            curl 'https://example.grafana.net/api/dashboards/home' -X 'Authorization: Bearer eyJrIjoiWWpNc1ZPS5RjdjdHTWlyVmtTcjI0Vk9xN1RkOGI1V3YiLCJuIjoieZl5b19vb30vb23iLCJpZCI6MX0='
+          containsSecret: true
+          match: eyJrIjoiWWpNc1ZPS5RjdjdHTWlyVmtTcjI0Vk9xN1RkOGI1V3YiLCJuIjoieZl5b19vb30vb23iLCJpZCI6MX0=
+        - text:
+            | # Some random text I found using GitHub search; decoded it is `{"k":"@LoadText", "v":"Loading..."}`.
+            curl 'https://not-a-dashboard.sonarsource.com' -d 'eyJrIjoiQExvYWRUZXh0IiwgInYiOiJMb2FkaW5nLi4uIn0='
+          containsSecret: false
+    - id: grafana-service-account-token
+      rspecKey: S6762
+      metadata:
+        name: Grafana service account tokens should not be disclosed
+      detection:
+        pre:
+          include:
+            content:
+              - glsa_
+        matching:
+          pattern: "\\b(glsa_[a-zA-Z0-9]{32}_[a-f0-9]{8})\\b"
```

Comment:
From a quick look, it seems `glsa_yscW25imSKJIuav8zF37RZmnbiDvB05G_fcaaf58a` is a well known test / example token. You might want to filter it out.

Reply:
I would also be in favor of adding a statistical filter. Recommended threshold 4.3.